erpnext_com/www/docs/user/manual/en/stock/articles/index.md

```diff
@@ -9,6 +9,7 @@
 
 ##### 2.2 Stock Reports
 1. [Stock Level Report](/docs/user/manual/en/stock/articles/stock-level-report)
+1. [Stock Value and Account Value Comparison](/docs/user/manual/en/stock/articles/stock-value-account-value-comparison)
```

Comment:
@rohitwaghchaure move this and previous item to /stock/file, these shouldn't be in /articles